### PR TITLE
Manage order

### PR DESCRIPTION
**Pull request by wiseemily88**
#### What does  this PR do?
Completes Store Admin can Manage orders: Index, and Show, and admin can update status for pending orders
#### Where should the reviewer start?
run test suite
#### Any background context you want to provide?
#153579498 see comment for adjustment to functionality- based on conversation with PM
#### What are the relevant story numbers?
#153579498
#### Questions:
  - Do Migrations Need to be ran? yes- added store_id to Order table and added orginal_price to Order item table.
  - Do Environment Variables need to be set? no
  - Any other deploy steps? no

### CODE REVIEW

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -2,8 +2,39 @@ class Admin::OrdersController < ApplicationController
 
   def index
```
Comment (aziobrow) [inline app/controllers/admin/orders_controller.rb:3]: looks like some merge conflicts--I'll take care of those here

File: app/views/admin/orders/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<<<<<<< HEAD
```
Comment (aziobrow) [inline app/views/admin/orders/index.html.erb]: resolving merge conflicts here too

File: app/views/admin/orders/show.html.erb [hunk 3]
```diff
@@ -37,9 +37,13 @@
   <p><b>Order Date:</b> <%= @order.date %><br />
   <b>Order Placed by:</b> <%= @order.user.full_name %><br />
```
Comment (aziobrow) [inline app/views/admin/orders/show.html.erb]: merge conflicts here

File: app/views/admin/orders/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <% @orders.each do |order| %>
+          <tr id="order-<%= order.id %>" class="order">
+            <th scope="row"> <%= link_to order.id,  admin_store_order_path(@store.url, order) %></th>
+            <td><%= order.date %></td>
+            <td><%= order.status.capitalize %></td>
+            <!-- <td>$<%= order.total_price %></td> -->
+            <td>
+              <% sum = 0 %>
+              <% order.items.each do |item| %>
+                <% sum += (item.price * OrderItem.find_by(item: item, order: order).quantity) %>
```
Comment (aziobrow) [inline app/views/admin/orders/index.html.erb:27]: we probably need a presenter for this...this is a lot of view logic

Comment (wiseemily88) [reply app/views/admin/orders/index.html.erb:27]: I agree- this is a chore ticket already on the icebox.

File: app/views/admin/orders/show.html.erb (hunk 3 above)
Comment (wiseemily88) [reply app/views/admin/orders/show.html.erb]: remove this part :  <b>Order Status:</b> <%= @order.status.capitalize %></p>

**Comment by aziobrow** [comment]
Whoo! Orders done!

**Pull request merged**